Sources/LogtoClient/LogtoAuthSession/LogtoWebViewAuthViewController.swift

```diff
@@ -25,9 +26,80 @@ public class LogtoWebViewAuthViewController: UnifiedViewController {
     override public func loadView() {
         view = webView
         webView.navigationDelegate = authSession
+        webView.configuration.userContentController.add(self, name: "socialHandler")
```

Comment:
`socialHandler` needs to align with both Android and iOS platforms.
Maybe we need an issue to track this.

Reply:
the idea is great, but `window.webkit` is more like an iOS stuff. not sure if we can align

Reply:
added LOG-2053, let's figure out this later